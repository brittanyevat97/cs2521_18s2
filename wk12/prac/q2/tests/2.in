from: 3 , to: 1
4 3 
2 5 
2 4
1 2 


 
