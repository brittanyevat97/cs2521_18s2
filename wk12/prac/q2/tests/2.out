hasPath, From: 3 , To: 1 

Edge inserted:  4 -> 3 
Edge inserted:  2 -> 5 
Edge inserted:  2 -> 4 
Edge inserted:  1 -> 2 


hasPath: 0 

